layouts/index.html

```diff
@@ -170,10 +170,11 @@ <h3 class="mt-5 text-white">Requirements</h3>
                     <p><strong>A.</strong> NVIDIA Volta™ or higher GPU with <a
                             href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 7.0+
                             <i class="fa-solid fa-arrow-up-right"></i></a></p>
-                    <p><strong>B.</strong> Ubuntu 20.04 or 22.04, Rocky Linux 8, or WSL2 on Windows 11</p>
+                    <p><strong>B.</strong> Linux or WSL2 on Windows 11</p>
```

Comment:
Maybe say something more like "A compatible Linux distro" and link to the list in the [new docs PR](https://github.com/rapidsai/docs/pull/559/files#diff-bead61f1a2dc63a4fb0ed77ea3c5e653c0625e31588439201f8bdd3c8ca83d67R135)?

Reply:
I like this idea! Thank you.

Reply:
```suggestion
                    <p><strong>B.</strong> <a href="https://docs.rapids.ai/install/#system-req" target="_blank">Compatible Linux distribution</a> or <a href="https://docs.rapids.ai/install#wsl2" target="_blank">WSL2 on Windows 11</a></p>
```